### DISCUSSION TITLE
Signature Change?

### DISCUSSION THREAD

**Discussion by rjm831** [j-fbriere/squawker#246] [Q&A] [answered]
The update for 3_7_7 is a different signature?

**Comment by j-fbriere** [comment, marked as answer]
No. 
The Squawker app from github (or from the alternate non official F-Droid type creativaxion repository) has one signature (that never changed).
The Squawker app from the official F-Droid repository has a different signature.
So if you had the app installed from the official F-Droid repository and then you tried to update it from github (or the alternate F-Droid type creativaxion  repository) - or vice versa - then you won't be able to do it.
You either keep the same repository source for your updates - or you do the following:
- Export all your data
- Uninstall the app
- Install the new version
- Import the data

**Reply by rjm831** [reply]
Thanks for the explanation.
